Neos.Kickstarter/Classes/Utility/Inflector.php

```diff
@@ -24,6 +24,7 @@ class Inflector
      */
     public function pluralize($word)
     {
+        /** @phpstan-ignore-next-line we could use autoloading instead */
```

Comment:
That comment doesn't explain anything to me… how would autoloading help with a static method call?

Reply:
the class is an ancient php library that we have in the resources folder and require_once via autoloading. Phpstan doesnt now any of it ^^

Reply:
Yep, instead of doing it in the Package.php we should just make this a https://getcomposer.org/doc/04-schema.md#files

Reply:
will try it with flow 9